state/factory.go

```diff
@@ -36,7 +36,6 @@ func (s *State) LoadConfigFromFile(filename string, networkFlag string) {
 	} else {
 		globals.Params.NetworkName = s.Network
 	}
-	fmt.Printf("\n\nNetwork : %s\n", s.Network)
```

Comment:
the network name already appear in many other places (notably the "Factom param" print)

Reply:
The factom param output only prints a non-default network name but not "MAIN". I think we should just make the default "network" param `"MAIN"` instead of `""` to address this.

Reply:
Sorry, actually the print comes from the state rather than the parameters: https://github.com/FactomProject/factomd/blob/FD-1225_wax_rollup/engine/NetStart.go#L101 so it doesn print MAIN